src/Installer/redist-installer/targets/BundledManifests.targets

```diff
@@ -31,12 +31,14 @@
       <RestoredNupkgContentPath>$(NuGetPackageRoot)$([MSBuild]::ValueOrDefault('%(NupkgId)', '').ToLower())/$([MSBuild]::ValueOrDefault('%(Version)', '').ToLower())</RestoredNupkgContentPath>
       <MsiNupkgId>%(Identity).Manifest-%(FeatureBand).Msi.$(MsiArchitectureForWorkloadManifests)</MsiNupkgId>
       <RestoredMsiNupkgContentPath>$(NuGetPackageRoot)$([MSBuild]::ValueOrDefault('%(MsiNupkgId)', '').ToLower())/$([MSBuild]::ValueOrDefault('%(Version)', '').ToLower())</RestoredMsiNupkgContentPath>
+      <IncludedInSource Condition="'%(IncludedInSource)' == ''">false</IncludedInSource>
+      <PackageDownload Condition="'$(DotNetBuildSourceOnly)' != 'true' or '%(IncludedInSource)' != 'true'">true</PackageDownload>
```

Comment:
Would it make sense to not specialize this for source-build?  e.g. the MSFT build would also use the version from the SDK repo.  It is best to avoid unnecessary differences between the two builds.  Keeping them the same helps flush out issues.

Reply:
Ok, I think this got past the pre-built errors now, but some of the smoke tests run into:
```
System.InvalidOperationException : Failed to execute /vmr/artifacts/bin/Microsoft.DotNet.SourceBuild.SmokeTests/Release/extracted-sdk/dotnet test /bl:/vmr/artifacts/TestResults/Release/BasicScenarioTests_MSTest_CSharp-test.binlog
Exit code: 1
/vmr/artifacts/bin/Microsoft.DotNet.SourceBuild.SmokeTests/Release/projects-202407231701323151/BasicScenarioTests_MSTest_CSharp/BasicScenarioTests_MSTest_CSharp.csproj : error MSB4242: SDK Resolver Failure: "The SDK resolver "Microsoft.DotNet.MSBuildWorkloadSdkResolver" failed while attempting to resolve the SDK "MSTest.Sdk/3.4.3". Exception: "System.IO.FileNotFoundException: Workload manifest microsoft.net.sdk.android: 34.99.0-preview.6.340/9.0.100-preview.6 from workload version 9.0.100-preview.7.24373.1 was not installed. Running "dotnet workload repair" may resolve this.
/vmr/artifacts/bin/Microsoft.DotNet.SourceBuild.SmokeTests/Release/projects-202407231701323151/BasicScenarioTests_MSTest_CSharp/BasicScenarioTests_MSTest_CSharp.csproj : error MSB4242:    at Microsoft.NET.Sdk.WorkloadManifestReader.SdkDirectoryWorkloadManifestProvider.GetManifests()
/vmr/artifacts/bin/Microsoft.DotNet.SourceBuild.SmokeTests/Release/projects-202407231701323151/BasicScenarioTests_MSTest_CSharp/BasicScenarioTests_MSTest_CSharp.csproj : error MSB4242:    at Microsoft.NET.Sdk.WorkloadManifestReader.WorkloadResolver.LoadManifestsFromProvider(IWorkloadManifestProvider manifestProvider)
/vmr/artifacts/bin/Microsoft.DotNet.SourceBuild.SmokeTests/Release/projects-202407231701323151/BasicScenarioTests_MSTest_CSharp/BasicScenarioTests_MSTest_CSharp.csproj : error MSB4242:    at Microsoft.NET.Sdk.WorkloadManifestReader.WorkloadResolver.InitializeManifests()
/vmr/artifacts/bin/Microsoft.DotNet.SourceBuild.SmokeTests/Release/projects-202407231701323151/BasicScenarioTests_MSTest_CSharp/BasicScenarioTests_MSTest_CSharp.csproj : error MSB4242:    at Microsoft.NET.Sdk.WorkloadManifestReader.WorkloadResolver.TryGetPackInfo(WorkloadPackId packId)
/vmr/artifacts/bin/Microsoft.DotNet.SourceBuild.SmokeTests/Release/projects-202407231701323151/BasicScenarioTests_MSTest_CSharp/BasicScenarioTests_MSTest_CSharp.csproj : error MSB4242:    at Microsoft.NET.Sdk.WorkloadMSBuildSdkResolver.CachingWorkloadResolver.Resolve(String sdkReferenceName, IWorkloadManifestProvider manifestProvider, IWorkloadResolver workloadResolver)
/vmr/artifacts/bin/Microsoft.DotNet.SourceBuild.SmokeTests/Release/projects-202407231701323151/BasicScenarioTests_MSTest_CSharp/BasicScenarioTests_MSTest_CSharp.csproj : error MSB4242:    at Microsoft.NET.Sdk.WorkloadMSBuildSdkResolver.CachingWorkloadResolver.Resolve(String sdkReferenceName, String dotnetRootPath, String sdkVersion, String userProfileDir, String globalJsonPath)
/vmr/artifacts/bin/Microsoft.DotNet.SourceBuild.SmokeTests/Release/projects-202407231701323151/BasicScenarioTests_MSTest_CSharp/BasicScenarioTests_MSTest_CSharp.csproj : error MSB4242:    at Microsoft.NET.Sdk.WorkloadMSBuildSdkResolver.WorkloadSdkResolver.Resolve(SdkReference sdkReference, SdkResolverContext resolverContext, SdkResultFactory factory)
/vmr/artifacts/bin/Microsoft.DotNet.SourceBuild.SmokeTests/Release/projects-202407231701323151/BasicScenarioTests_MSTest_CSharp/BasicScenarioTests_MSTest_CSharp.csproj : error MSB4242:    at Microsoft.Build.BackEnd.SdkResolution.SdkResolverService.TryResolveSdkUsingSpecifiedResolvers(IReadOnlyList`1 resolvers, Int32 submissionId, SdkReference sdk, LoggingContext loggingContext, ElementLocation sdkReferenceLocation, String solutionPath, String projectPath, Boolean interactive, Boolean isRunningInVisualStudio, SdkResult& sdkResult, IEnumerable`1& errors, IEnumerable`1& warnings)""
```
I'll see if I can reproduce this on GitHub codespaces.